mozilla/foundation.mozilla.org
# updated typos for highlight

Closes #6828


**Link to sample test page**: https://foundation-s-6828-renam-pgbdkl.herokuapp.com/cms/snippets/highlights/highlight/edit/1/

**Steps to test**:

1. Visit test page linked above [(slack review app thread)](https://mozilla.slack.com/archives/CBKP2SG94/p1639009430008100)
2. Double check the help text on the right that the typo is no longer present
3. If so, testing is complete!
